src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -97,9 +131,34 @@ public CommandResult addUserTags(Model model) throws CommandException {
         model.getObservablePersonList().forEach(person ->
                 person.setCommonModules(editedUser.getImmutableModuleTags()));
 
+        for (ModuleTag tag : moduleTags) {
```

Comment:
Same here.

Reply:
The for loop is fine, it’s only a problem if you want to put a double for loop for example.